packages/app-shortcut/ios/Plugin/AppShortcutPlugin.swift

```diff
@@ -6,13 +6,48 @@ import Capacitor
  * here: https://capacitorjs.com/docs/plugins/ios
  */
 @objc(AppShortcutPlugin)
-public class AppShortcutPlugin: CAPPlugin {
+public class AppShortcutPlugin: CAPPlugin, UIApplicationDelegate {
```

Comment:
I once read somewhere that you should only have one AppDelegate in an app. That's why I have the following implementation in the Firebase Messaging plugin: https://github.com/capawesome-team/capacitor-firebase/tree/main/packages/messaging#ios

And it seems to be the same for the official Push Notifications plugin: https://github.com/ionic-team/capacitor-plugins/blob/main/push-notifications/README.md#ios

Another example:  https://github.com/capawesome-team/capacitor-plugins/tree/main/packages/screen-orientation#ios